<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>688</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Network Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>WiFi Settings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>SSID</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ssid_input_">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="password_input_">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Devices</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Add new device</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>350</width>
                 <height>16777215</height>
                </size>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <property name="rightMargin">
                 <number>16</number>
                </property>
                <property name="verticalSpacing">
                 <number>10</number>
                </property>
                <item row="3" column="0">
                 <widget class="QToolButton" name="add_button_">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QWidget" name="widget_8" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>220</width>
                    <height>70</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>IP Address</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="device_ip_address_">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QWidget" name="widget_7" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>220</width>
                    <height>70</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Device Name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="device_name_input_">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QWidget" name="widget_9" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>220</width>
                    <height>70</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Port</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="device_port_">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="error_label_">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: red;</string>
               </property>
               <property name="text">
                <string>Error</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2"/>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Devices</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1">
              <widget class="QToolButton" name="delete_button_">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QTableWidget" name="table_widget_">
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column/>
               <column/>
               <column/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="action_buttons_widget_" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_button_">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
    background-color: #28a745;  /* Cor de fundo verde */
    color: white;  /* Cor do texto */
    border: none;
    border-radius: 5px;  /* Bordas arredondadas */
    padding: 10px 20px;  /* Espaçamento interno */
    font-size: 14px;
    font-weight: bold;
}

/* Estado hover - Salvar */
QPushButton:hover {
    background-color: #218838;  /* Cor mais escura quando passar o mouse */
}

/* Estado pressionado - Salvar */
QPushButton:pressed {
    background-color: #1e7e34;  /* Cor mais escura quando pressionado */
}
</string>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button_">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Botão de Cancelar - Vermelho */
QPushButton {
    background-color: #dc3545;  /* Cor de fundo vermelha */
    color: white;  /* Cor do texto */
    border: none;
    border-radius: 5px;  /* Bordas arredondadas */
    padding: 10px 20px;  /* Espaçamento interno */
    font-size: 14px;
    font-weight: bold;
}

/* Estado hover - Cancelar */
QPushButton:hover {
    background-color: #c82333;  /* Cor mais escura quando passar o mouse */
}

/* Estado pressionado - Cancelar */
QPushButton:pressed {
    background-color: #bd2130;  /* Cor mais escura quando pressionado */
}</string>
        </property>
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
